### PR TITLE
Jumpsuit Loadout Overlay Example (Not a real PR but look at it if you're a spriter)

### PR DESCRIPTION
**Pull request by Geevies**
Not a real PR, since I'm not a spriter, but this how to add a simple overlay system to jumpsuits that lets you add literally whatever on top of them.

Since jumpsuits are fairly boring at the moment, you can use this system to do things like putting corporate logos on the back, with the added customization of the colorable jumpsuit, to make some fairly unique designs. Additionally, you can also do something like adding department colors around the sleeves and bottom bits, which allows people to have departmental jumpsuits with clear markings, but in their own chosen color.

So basically, scavenge this PR if you manage to get sprites for it.

### CODE REVIEW

**Comment by alsoandanswer** [comment]
This could be definitely useful for the labcoats selection; most are the same except for the corporate logos on the back.

**Comment by atteria** [comment]
very interesting thanks
